build/ci-build.yml

```diff
@@ -23,16 +23,30 @@ resources:
 
 variables:
   - group: 'Build Configuration'
-  # Always use fixed version for .NET Core SDK
   - name: 'DotNet.Sdk.Version'
-    value: '2.2.402'
+    value: '3.0.101'
+  # Backwards compatible .NET SDK version
+  - name: 'DotNet.Sdk.VersionBC'
+    value: '2.2.105'
   - name: 'Project'
     value: 'Arcus.Templates'
   - name: 'Http.Port'
     value: '5000'
   - name: 'Arcus_Api_BaseUrl'
     value: 'http://localhost:$(Http.Port)/api/v1'
+  - name: 'Arcus_Worker_HealthPort'
+    value: '42065'
+  - name: 'Arcus_Worker_EventGrid_TopicUri'
+    value: 'https://arcustestingtopicname.westeurope-1.eventgrid.azure.net/api/events'
+  - name: 'Arcus_Worker_EventGrid_TopicName'
+    value: 'arcustestingtopicname' # We are re-using the Event Grid integration infrastructure
+  - name: 'Arcus_Worker_ServiceBus_QueueName'
+    value: 'arcustesting'
```

Comment:
Should be changed to Arcus indeed, maybe via an `Arcus Messaging - Integration Testing`

Reply:
Done.